scheduler.yaml_template

```diff
@@ -0,0 +1,127 @@
+default:
+    center: SMHI
```

Comment:
in the cfg file I call it `center_id`, so we should make it consistent by either changing in the cfg or in th yaml

Reply:
...or that's the way I wrote it in the documentation. So it has to be changed somewhere.

Reply:
ok, center_id is better.